OpenEdXMobile/res/layout/discussion_response_action_bar_layout.xml

```diff
@@ -1,17 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/discussion_actions_bar"
     style="@style/discussion_responses_nested_card_layout"
     android:layout_height="@dimen/discussion_responses_secondary_actions_bar_height"
     tools:showIn="@layout/row_discussion_responses_response">
 
+    <View
+        style="@style/light_gray_separator"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="MissingConstraints" />
+
     <LinearLayout
         android:id="@+id/discussion_responses_action_bar_vote_container"
         android:layout_width="wrap_content"
         android:layout_height="match_parent"
-        android:layout_gravity="start"
+        app:layout_constraintStart_toStartOf="parent"
```

Comment:
We can equally distribute the spacing between layout by adding the following code in all child `LinearLayout` of `ConstraintLayout`:

```
android:layout_width="0dp"
app:layout_constraintWidth_percent=".33"
```

it takes care the alignment, and equally distribution of width, after the code don't needs the following code in `TextView` style

```
<item name="android:maxWidth">@dimen/discussion_responses_action_bar_text_view_max_width</item>
```

Also set the gravity to  `android:gravity="center_vertical|start"`, `android:gravity="center"`, and `android:gravity="center_vertical|end"` respectively to make the Text align.